.env.dist

```diff
@@ -16,5 +16,5 @@ MYSQL_PASSWORD=
 MYSQL_DATABASE=
 
 ##### PHP
-PHP_IDE_CONFIG=serverName=
-XDEBUG_CONFIG=remote_host= idekey=PHPSTORM remote_port=9001 remote_enable=1 remote_connect_back=0
+PHP_IDE_CONFIG=
+XDEBUG_CONFIG=
```

Comment:
Why don't you keep the configuration, so we can easily switch if needed ? :)

Reply:
Because there is a dedicated page in the Wiki.

Reply:
Fair point.